README.markdown

```diff
@@ -1253,65 +1705,91 @@ $value = try_get_value($data, 'a/b/c/d', 'not_found')
 # using custom separator
 $value = try_get_value($data, 'a|b', [], '|')
 # $value = ['b1','b2','b3']
-~~~
+```
 
 1. **$data** The data structure we are working with.
 2. **'a/b/2'** The path string.
 3. **'not_found'** The default value. It will be returned if nothing is found.
-   (optional, defaults to *undef*)
+   (optional, defaults to *`undef`*)
 4. **'/'** The path separator character.
    (optional, defaults to *'/'*)
 
-#### `type3x`
+*Type*: rvalue.
+
+#### `type3x` TODO: is this deprecated at this point?
```

Comment:
It was deprecated when it was added, and documented as such:
- https://github.com/puppetlabs/puppetlabs-stdlib/pull/382/files#diff-67df06d66a84fa8e9b63ecf6dc90c761R7
- https://github.com/puppetlabs/puppetlabs-stdlib/pull/382/files#diff-0a369498a5a8db3ac8fa606b544c9810R467

It can be removed when we drop puppet 3, as stated. It doesn't have to print a warning to be deprecated.